Update droplet base image

hackerkid opened the pull request:
I accidentally deleted our current snapshot of the remote development environment while deleting a snapshot of `DO one click app`. So I generated a new snapshot after updating the base.zulipdev.org. The base droplet now has zulip/zulip commit till ac0d60f5773ade23ad61658f0349ca6df6493d55. I have also included the custom motd from b594708915ad4c2a1975860b9dc665eb98aa7d9a in base droplet.

We should also probably delete all other base.zulipdev.org snapshots as well. They are of no use currently and costs us money every month.
```bash
~/zulip on  droplet-update-4 [$] is 📦  v0.0.0 via ⬢ v8.11.3 took 7s 
➜ ssh zulipdev@hackerkid.zulipdev.org
Welcome to Ubuntu 16.04.1 LTS (GNU/Linux 4.4.0-47-generic x86_64)


This is the Zulip development environment.  Popular commands:
* tools/provision - Update the development environment
* tools/run-dev.py - Run the development server
* tools/lint - Run the linter (quick and catches many problems)
* tools/test-* - Run tests (use --help to learn about options)

Read https://zulip.readthedocs.io/en/latest/testing/testing.html to learn
how to run individual test suites so that you can get a fast debug cycle.

Last login: Fri Jun  7 14:43:07 2019 from 103.81.182.83
-bash: cd: /srv/zulip: No such file or directory
(zulip-py3-venv) ~$ 
```

timabbott commented:
This looks like some sort of bug:
`-bash: cd: /srv/zulip: No such file or directory`

Looking at the code, I think the actual problem is that `provision` behavior should be an option that we only use when provisioning a Vagrant development environment; otherwise /srv/zulip isn't guaranteed to be the right path.  Probably reasonable to just do a check for whether the username is "vagrant" for controlling that behavior of provision; can you take care of it?  I figure it's worth fixing before we merge this.

By the way, we should consider completely scripting the process for making a new base development droplet image; I'd imagine it's only a few commands?

Deleting the other base images sounds great to me; I hadn't realized they were running.  As a final note, accidentally deleting the wrong VM is a thing every programmer does at some point in life; that's one reason chat.zulip.org is in a different DO account.  We should think about whether there's other things we can do with e.g. naming (or potentially making a DO organization entirely for the prod base images) to make sure it isn't a mistake we repeat.

hackerkid commented:
@timabbott 

* Added a commit to fix the provision issue.
* Deleted all the old snapshot images of base.zulipdev.org
* Added to my TODO list to write a Fabric script for automating base droplet upgrades and image creation.

timabbott commented:
Merged, thanks @hackerkid!  

For that last follow-up, I'm not sure whether we want to do a Fabric script or just a shell script; I would sorta guess a shell script would be simpler.
